CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Welcome
+
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```

Comment:
This isn't an 18F open source project.

Reply:
Good point. @afeld what do we need to do for the CONTRIBUTING.md file?

Reply:
I had been using the 18F one out of laziness, but the template to use is actually [the GSA one](https://github.com/GSA/open-source-policy/blob/master/CONTRIBUTING.md).